### ISSUE TITLE
[Feature request] Add option to remove "Translate X to English" in text highlight

### ISSUE THREAD

**Issue by IAMNOTRANA** [stonecrusher/simpleMenuWizard#92]
![1](https://github.com/user-attachments/assets/8a6787da-9e36-42ff-bcec-f8f4ced8bee3)
New Firefox update 128.0 added "Translate" option in context menu.

**Comment by Meladoom** [comment]
Having less addons is a good thing, but TWP has way more languages, so this new thing isn't really useful... yet

Also it uses website language instead of selected text. Really annoying for multi-language sites

**Comment by F4400** [comment]
After a bit of testing: `#context-translate-selection`
This should be it.

**Comment by IAMNOTRANA** [comment]
> After a bit of testing: `#context-translate-selection` This should be it.

Well, that didnt work, but I manage to disable it.

Disable "browser.translations.enable" in "about:config" and it should get rid of all translation option.

**Comment by IAMNOTRANA** [comment]
> > After a bit of testing: `#context-translate-selection` This should be it.
> 
> Well, that didnt work, but I manage to disable it.
> 
> Disable "browser.translations.enable" in "about:config" and it should get rid of all translation option.

**Comment by stonecrusher** [comment]
`#context-translate-selection` does work for me, though.

@F4400 Did you test it?

I'm sorry that I'm a bit late. Will update it into the repo soon.

**Comment by F4400** [comment]
Ja, works for me and I am currently using it; don't know why it doesn't work for IAMNOTRANA.

**Comment by stonecrusher** [comment]
Thanks for the fast response. So I will add it.
